Sources/Sentry/SentryFileManager.m

```diff
@@ -611,7 +611,7 @@ - (NSNumber *_Nullable)readTimezoneOffset
 
 - (void)storeTimezoneOffset:(NSInteger)offset
 {
-    NSString *timezoneOffsetString = [NSString stringWithFormat:@"%zd", offset];
+    NSString *timezoneOffsetString = [NSString stringWithFormat:@"%ld", (long)offset];
```

Comment:
Please add a test case for this change.

Reply:
There is an existing test case `testStoreAndReadTimezoneOffset`, but i added another test case that does negative numbers.

Reply:
Sorry, I should have been more specific. You change the code for a reason cause it doesn't work for a specific use case. It would be great to add a test that would fail when using `@"%zd"`, but passes when using `@"%ld"`. I guess that's what you did with the extra test case.

Reply:
Since the API that works with the timezone offset all restrict it to NSInteger, it's not possible to overflow the value that'll ultimately be passed to this. I think the tests are good at this point.